tests/unit/test_handlers.py

```diff
@@ -804,3 +807,77 @@ def test_add_md5_with_bytes_object(self):
         self.assertEqual(
             request_dict['headers']['Content-MD5'],
             'OFj2IjCsPJFfMAxmQxLGPw==')
+
+
+class TestParameterAlias(unittest.TestCase):
```

Comment:
I would say the expected result would be failure on the old without any special ambiguity.